src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,105 @@
+import { Link } from 'react-router-dom';
+
+const NavBar = () => {
+    return (
+        <div>
+            <nav className="nav-main flex flex-wrap justify-between items-center p-4 border-b-2 text-lg h-16">
+                {/*  Nav left */}
+                <ul className="nav-main-left nav-main-ul hover:cursor-pointer">
+                    <li className="nav-main-li nav-section-one">
+                        <Link to="/" className="nav-main-item nav-item-home">
+                            TI <span className="text-color-highlight-orange">Social</span>
+                        </Link>
+                        {/* <i
+                            className="far fa-bars hamburger-menu"
+                            onClick={() => toggleHamburger()}
+                        ></i> */}
```

Comment:
you can remove this I guess as it's a LInk now

Reply:
This will be the menu button once I make it responsive. Just commented it out for now